app/scripts/directives/mobileClientServiceAssociations.js

```diff
@@ -40,7 +40,21 @@
       resource: 'mobileclients'
     };
 
-    ctrl.$onInit = function () {
+    var updateExcluded = function(mobileClient) {
```

Comment:
`updateAssociations`? as it updates both lists?
May as well attach this to the controller as it makes sense as a controller method.